apps/react-vite/vite.config.ts

```diff
@@ -4,10 +4,18 @@
 import react from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
 import viteTsconfigPaths from 'vite-tsconfig-paths';
+import { Schema, ValidateEnv } from '@julr/vite-plugin-validate-env';
 
 export default defineConfig({
   base: './',
-  plugins: [react(), viteTsconfigPaths()],
+  plugins: [
+    react(),
+    viteTsconfigPaths(),
+    ValidateEnv({
```

Comment:
Hey, thanks for making this PR. I think we can pass Zod schema here from `src/config/env`, to make it consistent...